riscv-configuration-structure-draft.adoc

```diff
@@ -268,9 +268,37 @@ big immediate benefit of a format that supports comments.
 [[sec:AccessMethod]]
 == Access Method
 
-The binary configuration structure is memory-mapped in system memory.
-There is a CSR which contains the physical address where the structure
-starts.
+The binary configuration structure is memory-mapped in system memory.  There is
+a CSR which contains the physical address where the structure starts.
+
+TODO: Get a CSR number assigned.
+
```

Comment:
What is mipmid? I can't find it mentioned in the privspec. There is mimpid but that already has a definition that's not compatible with it being a pointer.

Reply:
Typo, it should be **mimpid**. 
Although it already has the definition, we still can have a specific mechanism to get the memory map address using mimpid. For example, write 55aa55aa to mimpid, then read mimpid that returns the memory map address. Then write another 55aa55aa to close the session. This way we don't have to introduce a new CSR or the exception to probe the existence of that CSR.

Reply:
That seems relatively complex. In addition, mimpid has number 0xF13 which means it's in the read-only space of CSRs so should not be writable.

But it's a good point that it would be nice to avoid catching exceptions. Looking through the privileged spec more closely, there's mconfigptr which appears to be already set aside for a binary structure. "mconfigptr must be implemented, but it may be hardwired to zero to indicate the configuration data structure does not exist or that an alternative mechanism must be used to locate it." So that's perfect. I'll update the PR to reference mconfigptr.